crates/host/src/wasmbus/mod.rs

```diff
@@ -3727,6 +3780,30 @@ fn ensure_actor_capability(
     Ok(())
 }
 
+fn injector_to_headers(injector: &OtelHeaderInjector) -> HeaderMap {
```

Comment:
```suggestion
fn injector_to_headers(injector: OtelHeaderInjector) -> HeaderMap {
```

Looking at the usages of this, you're always creating the value and then immediately converting it, so it seems that at least this should consume the value